digger/builds/cordova_android.py

```diff
@@ -56,3 +58,51 @@ def get_export_path(self):
     Gets the apk(s) path that can be exported outside of the container.
     """
     return ','.join(android_helper.find_apks(self.path))
+
+
+class CordovaLightAndroidBuild(CordovaAndroidBuild):
+  def __init__(self, tmp_folder=None, *args, **kwargs):
+    super(CordovaLightAndroidBuild, self).__init__(*args, **kwargs)
+    self.tmp_folder = tmp_folder
+    if self.tmp_folder is None:
+      self.tmp_folder = '%s_tmp' % self.path.split('/')[-1:][0]
```

Comment:
I think we can send another PR to handle relative path (including using the current one if --path is not used)